rclcpp/include/rclcpp/node.hpp

```diff
@@ -127,7 +128,9 @@ class Node
     size_t queue_size,
     std::function<void(const std::shared_ptr<MessageT> &)> callback,
     rclcpp::callback_group::CallbackGroup::SharedPtr group = nullptr,
-    bool ignore_local_publications = false);
+    bool ignore_local_publications = false,
+    typename rclcpp::msg_memory_strategy::MessageMemoryStrategy<MessageT>::SharedPtr
+    msg_mem_strat = rclcpp::msg_memory_strategy::MessageMemoryStrategy<MessageT>::create_default());
```

Comment:
(IMO) I think Uncrustify did the right thing there. No different than this:

```c++
template<...>
void
func(Type param);
```

There are plenty of examples of namespace escalation, e.g. `rclcpp::Node` and `rclcpp::node::Node` are both valid, so you could raise the `MessageMemoryStrategy` into the `rclcpp` namespace. Though that doesn't typically happen outside of convenience headers for the user, like `rclcpp/rclcpp.hpp`.